File: src/Suframa.Sciex.UserInterface/www/app/view-model/CalendarioAgendamentoVM.ts
import { calendarioDiaVM } from './CalendarioDiaVM';
import { calendarioHoraVM } from './CalendarioHoraVM';
import { usuarioInternoVM } from './UsuarioInternoVM';

export class calendarioAgendamentoVM {
	constructor() {
		this.analistas = new Array<usuarioInternoVM>();
		this.dias = new Array<calendarioDiaVM>();
		this.horarios = new Array<calendarioHoraVM>();
	}

	analistas: usuarioInternoVM[];
	ano?: number;
	dias: calendarioDiaVM[];
	calendario: any[];
	horarios: calendarioHoraVM[];
	idCalendarioAgendamento?: number;
	idUnidadeCadastradora?: number;
	idUsuarioInterno?: number;
	isMesInteiro: boolean;
	mes?: number;
}

File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/CalendarioAgendamentoVM.cs
using System;
using System.Collections.Generic;

namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
	public class CalendarioAgendamentoVM
	{
		public IEnumerable<UsuarioInternoVM> Analistas { get; set; }
		public int? Ano { get; set; }
		public IEnumerable<CalendarioVM> Calendario { get; set; }
		public DateTime? DataAtendimento { get; set; }
		public IEnumerable<CalendarioDiaVM> Dias { get; set; }
		public IEnumerable<CalendarioHoraVM> Horarios { get; set; }
		public int? IdCalendarioAgendamento { get; set; }
		public int? IdUnidadeCadastradora { get; set; }
		public int? IdUsuarioInterno { get; set; }
		public bool IsMesInteiro { get; set; }
		public int? Mes { get; set; }
		public string UnidadeCadastradora { get; set; }
	}
}